meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
can this be common file  under meta-cmf-broadband

Reply:
I think yes we can make it common file under meta-cmf-broadband, in that case simultaneously  we need to remove same changes from meta-cmf-raspberrypi.

Reply:
ok